packages/blue-sdk-viem/src/fetch/Position.ts

```diff
@@ -22,20 +23,21 @@ export async function fetchPosition(
   parameters.chainId ??= await getChainId(client);
 
   const { morpho } = getChainAddresses(parameters.chainId);
-  const [supplyShares, borrowShares, collateral] = await readContract(client, {
-    ...parameters,
-    address: morpho,
-    abi: blueAbi,
-    functionName: "position",
-    args: [marketId, user],
-  });
+  const position = restructure(
+    await readContract(client, {
+      ...parameters,
+      address: morpho,
+      abi: blueAbi,
+      functionName: "position",
+      args: [marketId, user],
+    }),
+    { abi: blueAbi, name: "position", args: ["0x", "0x"] },
```

Comment:
The thing that bothers me is that the dev has to pass dummy args to identify the abi item

Looking at viem's code, it makes sense: you have to make sure that the arg type matches the input types of the abi item, because 2 abi items can have the same name but not the same inputs

But it was non trivial for me to understand why this param is required and what to put in here when using it

Maybe defining dummy args exported from utils like this could help:
```ts
export const addressArg: Address = "0x";
export const hexArg: Hex = "0x";
export const bigintArg: bigint = 0n;
export const intArg: number = 0;
export const stringArg: string = "";
```

I built this reverse registry based on abitype's registry used by viem in their `AbiParameter` type